drivers/serial/uart_nrfx_uarte.c

```diff
@@ -856,8 +856,8 @@ static void notify_uart_rx_rdy(const struct device *dev, size_t len)
 	user_callback(dev, &evt);
 }
 
-static void notify_rx_buf_release(const struct device *dev,
-				  uint8_t **buf, bool clear)
+static void rx_buf_release(const struct device *dev,
+				  uint8_t **buf)
```

Comment:
Now whole the function signature will fit in a single line. This will also fix the indentation here.

Reply:
Right, done